agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -948,3 +949,69 @@ func testV3TaskEndpointTags(t *testing.T, taskName, containerName, networkMode s
 	exitCode, _ := task.ContainerExitcode(containerName)
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
+
+func TestContainerOrdering(t *testing.T) {
+	agent := RunAgent(t, nil)
+	defer agent.Cleanup()
+	agent.RequireVersion(">=1.26.0")
```

Comment:
Thanks Eric. I have made it >=1.25.0

Reply:
Please keep this at 1.26.0. This won't run on 1.25.0 (since we are at 1.25.3 + changes).  Its true this won't run if you build without updating the version file -- but we will do just that when we go to do the next release.